### ISSUE TITLE
Invalid client code generated when using JSON schema

### ISSUE THREAD

**Issue by stevenroose** [Jumpscale/go-raml#242]
I have a RAML file that references a JSON schema file like this:
```raml
types:
  MyType:
    description: [...]
    type: !include schemas/mytype.json
```

I asked my employer if I'm allowed to share the JSON schema, it shouldn't be a problem, so I'll get back and put it here once he responds. The schema passes online linters and is also interpreted correctly by [gojsonschema](https://github.com/xeipuuv/gojsonschema).

I'm pretty sure at least a part of it is related to regex patterns. The generated Go code seems to have some wrong raw JSON in it.

**Comment by iwanbk** [comment]
Hi @stevenroose, thanks for the report.

go-raml should support JSON schema. We have test case for this in https://github.com/Jumpscale/go-raml/blob/master/codegen/fixtures/struct/json/api.raml#L5, generated code is https://github.com/Jumpscale/go-raml/tree/master/codegen/golang/fixtures/struct/json

```
I'm pretty sure at least a part of it is related to regex patterns. The generated Go code seems to have some wrong raw JSON in it.
```
Can you post the fields that use regex?
we also have another issue with regex in #132

**Comment by stevenroose** [comment]
This is the JSON schema file:
https://gist.github.com/stevenroose/c967c273c67daa58193fc0b425fcc580#file-mt103-json

[Above it](https://gist.github.com/stevenroose/c967c273c67daa58193fc0b425fcc580#file-mt103-go), there is the Go file that is created.

**Comment by iwanbk** [comment]
I tested it by:
- replacing https://github.com/Jumpscale/go-raml/blob/master/codegen/fixtures/struct/json/person.json with your json.
- use master
- using this command `go-raml client --dir /tmp/ehem --ramlfile codegen/fixtures/struct/json/api.raml `

The code looks good, at least it was compiled.
```
package client

import (
    "gopkg.in/validator.v2"
)

// A JSON schema for the MT103 SWIFT message.
type PersonInclude struct {
    F13C string                `json:"F13C" validate:"regexp=^/[A-Z]{0,8}/[0-9]{4}[-+][0-9]{4}$,nonzero"`
    F20  string                `json:"F20" validate:"max=16,nonzero"`
    F23B EnumPersonIncludeF23B `json:"F23B" validate:"nonzero"`
    F23E string                `json:"F23E" validate:"regexp=^(CHQB|CORT|HOLD|INTC|PHOB|PHOI|PHON|REPA|SDVA|TELB|TELE|TELI)(/.{1,30})?$,nonzero"`
    F26T string                `json:"F26T" validate:"min=3,max=3,nonzero"`
    F32A string                `json:"F32A" validate:"max=24,regexp=^[0-9]{6}[A-Z]{3}[0-9]{1,15}(,[0-9]{1,13})?$,nonzero"`
    F33B string                `json:"F33B" validate:"max=18,regexp=^[A-Z]{3}[0-9]{1,15}(,[0-9]{1,13})?$,nonzero"`
    F36  string                `json:"F36" validate:"max=12,regexp=^[0-9]{1,12}(,[0-9]{1,10})$,nonzero"`
    F50a string                `json:"F50a" validate:"nonzero"`
    F51A string                `json:"F51A" validate:"nonzero"`
    F52a string                `json:"F52a" validate:"nonzero"`
    F53a string                `json:"F53a" validate:"nonzero"`
    F54a string                `json:"F54a" validate:"nonzero"`
    F55a string                `json:"F55a" validate:"nonzero"`
    F56a string                `json:"F56a" validate:"nonzero"`
    F57a string                `json:"F57a" validate:"nonzero"`
    F59a string                `json:"F59a" validate:"nonzero"`
    F70  string                `json:"F70" validate:"nonzero"`
    F71A EnumPersonIncludeF71A `json:"F71A" validate:"nonzero"`
    F71F string                `json:"F71F" validate:"max=18,regexp=^[A-Z]{3}[0-9]{1,15}(,[0-9]{1,13})$,nonzero"`
    F71G string                `json:"F71G" validate:"max=18,regexp=^[A-Z]{3}[0-9]{1,15}(,[0-9]{1,13})$,nonzero"`
    F72  string                `json:"F72" validate:"nonzero"`
    F77B string                `json:"F77B" validate:"nonzero"`
}

func (s PersonInclude) Validate() error {

    return validator.Validate(s)
}
```

**Comment by stevenroose** [comment]
Let me try again

**Comment by iwanbk** [comment]
@stevenroose sorry for late reply, was on vacation.

I can reproduce it now.

your file
```
MT103:
    description: ...
    type: !include schemas/mt103.json
```
my test
```
MT103: !include schemas/mt103.json
```

**Comment by iwanbk** [comment]
@stevenroose fixed in #248

**Comment by stevenroose** [comment]
Awesome, thanks!
